src/main/groovy/net/foragerr/jmeter/gradle/plugins/TaskJMInit.groovy

```diff
@@ -95,7 +99,7 @@ class TaskJMInit extends DefaultTask {
                 writer.flush();
                 writer.close();
             } catch (IOException ioe) {
-                throw new GradleException("Couldn't create temporary property file " + f.getName() + " in directory " + workDir.getPath(), ioe);
+                throw new GradleException("Couldn't create temporary property file " + f.getName() + " in directory " + project.jmeter.workDir.getPath(), ioe);
```

Comment:
You seem to've generally replaced `workDir` with `binDir`, why is this particular line back to `workDir`?